docs/guides/ADAPTER-DEVELOPMENT.md

```
@@ -0,0 +1,320 @@
+  });
+  assert.ok(Array.isArray(config.files));
+});
+
+void test("windsurf workspace --help shows windsurf-specific help", async () => {
+  // Use the built-cli-harness to verify CLI integration
+});
+```
+
+### 13. Update Documentation
+
+- Add your host to the README [Supported hosts](#supported-hosts) badge row
```

markdownlint-cli2 (0.23.1)

[warning] Link fragments should be valid

(MD051, link-fragments)